packages/squiggle-lang/src/rescript/Reducer/Reducer_Expression/Reducer_Expression.res

```diff
@@ -0,0 +1,90 @@
+module BuiltIn = Reducer_Dispatch_BuiltIn
+module ExpressionValue = ReducerInterface.ExpressionValue
+module Extra = Reducer_Extra
+module MathJs = Reducer_MathJs
+module Result = Belt.Result
+module T = Reducer_Expression_T
+open Reducer_ErrorValue
+
+type expression = T.expression
```

Comment:
Related, I could see calling this ``t`` and treating functions accordingly. Less sure here though.

Reply:
It is a module name. Type name would be t.

Reply:
module Reducer.Expression
    type t = Reducer.Expression.T
would be your small t properly

Reply:
I defined small t

Reply:
Yea; I'm suggesting doing exactly that. (Writing ``type t = Reducer.Expression.T.expression`` in this file). 

When you said, "I defined small t"; I don't understand? It seems like this still isn't done in this file.

Reply:
(Wait, I see this has been done right under this, at this point, will remove)

Reply:
RESOLVED